modules/pool/lib/pool-gql-loader.service.ts

```diff
@@ -154,8 +158,15 @@ export class PoolGqlLoaderService {
         return prisma.prismaPool.count({ where: this.mapQueryArgsToPoolQuery(args).where });
     }
 
-    public async getFeaturedPoolGroups(): Promise<GqlPoolFeaturedPoolGroup[]> {
-        const featuredPoolGroups = await networkContext.config.contentService.getFeaturedPoolGroups();
+    public async getFeaturedPoolGroups(chains: Chain[]): Promise<GqlPoolFeaturedPoolGroup[]> {
+        const featuredPoolGroups = [];
+        if (chains.some((chain) => BalancerChainIds.includes(chainToIdMap[chain]))) {
+            const githubContentService = new GithubContentService();
+            featuredPoolGroups.push(...(await githubContentService.getFeaturedPoolGroups(chains)));
+        } else if (chains.some((chain) => BeethovenChainIds.includes(chainToIdMap[chain]))) {
+            const sanityContentService = new SanityContentService('FANTOM');
+            featuredPoolGroups.push(...(await sanityContentService.getFeaturedPoolGroups(chains)));
+        }
         const poolIds = featuredPoolGroups
```

Comment:
can we remove if / else and just query both? they will return empty array if there is no pools for a chain, right?